demisto_sdk/commands/validate/tests/DO_validators_test.py

```diff
@@ -80,20 +69,506 @@ def test_DockerImageExistValidator_is_valid(
+            values=[
+                "LatestDockerImageIntegrationScript",
+                "demisto/python3:latest",
+            ],  # script with 'latest' as the docker tag
+        ),
+        create_integration_object(),  # integration without 'latest' docker tag
+        create_script_object(),  # script without 'latest' docker tag
+        create_integration_object(
+            paths=["script.type"], values=["javascript"]
+        ),  # javascript integration
+        create_script_object(
+            paths=["type"], values=["javascript"]
+        ),  # javascript script
+    ]
+    results = LatestDockerImageTagValidator().is_valid(content_items)
+    assert len(results) == 2
+    for result in results:
+        content_item: IntegrationScript = result.content_object
+        assert content_item.type == "python"
+        assert content_item.docker_image == "demisto/python3:latest"
+        assert content_item.name == "LatestDockerImageIntegrationScript"
+
+
+def test_DockerImageIsNotDemistoValidator_is_valid():
+    """
+    Given:
+     - 1 integration and 1 script which uses a valid demisto version
+     - 1 integration and 1 script which do not use a valid demisto version
+     - 1 integration and 1 script that are javascript
+
+    When:
+     - Running the DockerImageIsNotDemistoValidator validator
+
+    Then:
+     - make sure the integrations/scripts that are not valid demisto images fail
+     - make sure that javascripts integrations/scripts are ignored
+    """
+    from demisto_sdk.commands.validate.validators.DO_validators.DO101_docker_image_is_not_demisto import (
+        DockerImageIsNotDemistoValidator,
+    )
+
+    content_items = [
+        create_integration_object(
+            paths=["name", "script.dockerimage"],
+            values=[
+                "NonDemistoImageIntegrationScript",
+                "repository/python3:latest",
+            ],  # integration with non-demisto image
+        ),
+        create_script_object(
+            paths=["name", "dockerimage"],
+            values=[
+                "NonDemistoImageIntegrationScript",
+                "repository/python3:latest",
+            ],  # script with non-demisto image
+        ),
+        create_integration_object(),  # integration with demisto image
+        create_script_object(),  # script with demisto image
+        create_integration_object(
+            paths=["script.type"], values=["javascript"]
+        ),  # javascript integration
+        create_script_object(
+            paths=["type"], values=["javascript"]
+        ),  # javascript script
+    ]
+    results = DockerImageIsNotDemistoValidator().is_valid(content_items)
+    assert len(results) == 2
+    for result in results:
+        content_item: IntegrationScript = result.content_object
+        assert content_item.type == "python"
+        assert content_item.docker_image == "repository/python3:latest"
+        assert content_item.name == "NonDemistoImageIntegrationScript"
```

Comment:
Same here and for all other tests, let's validate the message.

Reply:
see my comment [here](https://github.com/demisto/demisto-sdk/pull/3899#discussion_r1446065790)